app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt

```diff
@@ -246,8 +254,6 @@ class QuestionPlayerActivityTest {
     }
   }
 
-  // TODO(#2057): Remove when TextViews are properly measured in Robolectric.
-  @RunOn(TestPlatform.ESPRESSO) // Incorrectly passes on Robolectric and shouldn't be re-enabled
```

Comment:
Why is this TODO removed? #2057 is still open.

Reply:
Sorry I was mistaken i thought since all the tests passed on roboelectric so there was no need for the Todos but now I see the issue here still exists.

Reply:
Readding the todos